lib/SearchAndSort/components/DateRangeFilter/DateRangeFilter.js

```diff
@@ -164,9 +164,9 @@ const TheComponent = ({
     };
 
     setFilterValue({
-      ...defaultFilterState,
+      ...filterValue,
       errors: {
-        ...defaultFilterState.errors, // reset filter errors on every change
+        ...filterValue.errors, // reset filter errors on every change
```

Comment:
Now the current state of `filterValue` is used, so we can remove the "reset filter errors on every change" comment.